src/components/AvatarWithImagePicker.js

```diff
@@ -155,6 +155,7 @@ class AvatarWithImagePicker extends React.Component {
     }
 
     render() {
+        // eslint-disable-next-line react/destructuring-assignment
         const {DefaultAvatar} = this.props;
```

Comment:
Could also change this to

```js
const DefaultAvatar = this.props.DefaultAvatar;
```

Reply:
Went through and removed all the exceptions made for component-type props to use your suggestion here 👍